lib/KIcon/iconDefinitions.js

```diff
@@ -0,0 +1,268 @@
+  // TopNavbar
+  dashboard: { icon: require('./material-svg/dashboard/Baseline.vue').default },
+  insert_chart: { icon: require('./material-svg/insert_chart/Baseline.vue').default },
+  // DeviceTopNav
+  https: { icon: require('./material-svg/https/Baseline.vue').default },
+  perm_device_information: {
+    icon: require('./material-svg/perm_device_information/Baseline.vue').default,
+  },
+  settings: { icon: require('./material-svg/settings/Baseline.vue').default },
+  // SettingsButton
+  tune: { icon: require('./material-svg/tune/Baseline.vue').default },
+  // TocButton
+  list: { icon: require('./material-svg/list/Baseline.vue').default },
+  // TopBar (and likely others?)
+  fullscreen: { icon: require('./material-svg/fullscreen/Baseline.vue').default },
+  fullscreen_exit: { icon: require('./material-svg/fullscreen_exit/Baseline.vue').default },
+  //SyncInterface
+  verified_user: { icon: require('./material-svg/verified_user/Baseline.vue').default },
+  //FacilityTopNav
+  save: { icon: require('./material-svg/save/Baseline.vue').default },
+  domain: { icon: require('./material-svg/domain/Baseline.vue').default },
+  // LearnTopNav
+  forum: { icon: require('./material-svg/forum/Baseline.vue').default },
+  // Searchbox
+  filter_list: { icon: require('./material-svg/filter_list/Baseline.vue').default },
+  warning: { icon: require('./material-svg/warning/Baseline.vue').default },
+};
+
+/**
+ * Given one of `blackPaths` or `coloredPaths` from `allIcons`,
+ * returns array where each element is an array of aliases which
+ * refer to the same icon. The sub-arrays are already sorted
+ * alphabetically.
+ *
+ * Example:
+ *
+ * [
+ *     ['arrow_back', 'back'],
+ *     ['arrow_forward', 'forward'],
+ *     ...
+ *     ['verified_user']
+ * ]
+ *
+ */
+
+const sortedIcons = paths => {
+  return Object.keys(paths)
+    .map(a => paths[a])
+    .sort((a, b) => a[0].charCodeAt(0) - b[0].charCodeAt(0));
+};
+
+/*
+ * Assigned the value: `{ blackPaths, coloredPaths }` from the function below.
+ *
+ * `blackPaths` and `coloredPaths` are objects wherein the keys are unique `paths` to an icon
+ *
+ * These path keys map to an array of alphabetically sorted aliases associated with that path.
+ */
+
+const allIcons = (() => {
+  let blackPaths = {};
+  let coloredPaths = {};
+
+  Object.keys(KolibriIcons).forEach(alias => {
+    const path = KolibriIcons[alias].icon.__file;
+    const hasColorToken = KolibriIcons[alias].defaultColor;
+    const thePathsWeCareAbout = hasColorToken ? coloredPaths : blackPaths;
+
+    let aliases = [alias];
+
+    if (Object.keys(thePathsWeCareAbout).includes(path)) {
+      aliases = [...thePathsWeCareAbout[path], ...aliases].sort();
+    }
+
+    thePathsWeCareAbout[path] = aliases;
+
+    if (hasColorToken) {
+      coloredPaths = thePathsWeCareAbout;
+    } else {
+      blackPaths = thePathsWeCareAbout;
+    }
+  });
+
+  return { blackPaths, coloredPaths };
+})();
```

Comment:
might be able to use ES2015 classes for this?

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes